plugins-builtin/extractcode-libarchive-system_provided/LICENSE.txt

```diff
@@ -0,0 +1,143 @@
+This is the primary license for this library.
```

Comment:
Is this file redundant with  plugins-builtin/extractcode-libarchive-system-provided/LICENSE.txt  https://github.com/nexB/scancode-toolkit/pull/1706/files#diff-22dc5702b8752ccfa9265468d79e3032 ?

Reply:
Yes!